commit 969118fae513afb075640258d89a5c5486476f19 (refs/changes/98/4598/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-07-27T14:06:11-07:00 (1 year, 2 months ago)
    
    TRITON-635 restify-clients usage should use 'contentMd5' option to avoid cross-node version BadDigest errors

diff --git a/package.json b/package.json
index a63c0a2..71ff137 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "workflow",
   "description": "SmartDataCenter Workflow API and Runners",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "repository": {
@@ -15,7 +15,7 @@
     "imgmanifest": "git+https://github.com/joyent/node-imgmanifest.git#e21ea53",
     "lstream": "0.0.4",
     "restify": "4.1.1",
-    "sdc-clients": "10.5.0",
+    "sdc-clients": "11.3.1",
     "uuid": "3.0.1",
     "vasync": "1.6.4",
     "verror": "1.6.0",
